### PR TITLE
Popout window

### PR DESCRIPTION
**Pull request by Daniel-Kigozi**
-Add a pop-up to mobile and desktop

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -374,6 +374,96 @@
...
+    display: flex;
+    flex-direction: row;
+    padding-inline-start: 0;
+  }
+
+  .pop-btn {
+    margin: 5px;
+    background-color: #e7e7e7;
+    border-radius: 5px;
+  }
```
Comment (m05e5) [inline styles.css:433]: - [ ] Hi to improve the UI ill recommend giving your `pop-btn` padding to centralize the text.

File: mobile.js [hunk 2]
```diff
@@ -1,10 +1,101 @@
...
+            <li class="pop-btn"> <a class="button-pop" type="button" href="">${technologies[0]}</a></li>
+            <li class="pop-btn"> <a class="button-pop" type="button" href="">${technologies[2]}</a></li>
+            <li class="pop-btn"> <a class="button-pop" type="button" href="">${technologies[3]}</a></li>
+        </ul>
+    </div>
+    <p class="pop-desc">${description}</p>
+    <div class=main-works-pop>
+        <a class="button button-line" type="button" href="${LiveLink}">See Live <span><img src="media/github.svg"/></span></a>
+        <a class="button button-line" type="button" href="${SourceLink}">See Source <span><img id="img-pop" src="media/github.svg"/></span></a>  
+    </div>
```
Comment (m05e5) [inline mobile.js]: Please you're required to follow the provided design as much as possible
- [ ] the icon on the `see live` should be the same as on FIGMA
- [ ] the icon's color should be white as on Figma
- [ ] the `main-works-pop` should be at the right side of the pop-up just below the image

File: styles.css [hunk 3]
```diff
@@ -374,6 +374,96 @@
...
+    background-color: rgba(0, 0, 0, 0.5);
+    backdrop-filter: 8px;
+    opacity: 0;
+    visibility: hidden;
+  }
+
+  .popup-img {
+    height: 30%;
+    margin: auto;
+  }
```
Comment (m05e5) [inline styles.css:406]: - [ ] the image should be more than 90% the with of the popup

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Daniel-Kigozi , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights

- all linters are green

## Required Changes ♻️
* You really did a great job but there are some features you need to improve

**your UI (Desktop)**
image

**Figma UI**
image

- [ ] Please see the comments below

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Daniel-Kigozi 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done on making the requested changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**